src/alpha_norm_type2.js

```diff
@@ -0,0 +1,11 @@
+export default function alpha_norm_type2(x1, x2, alpha) {
```

Comment:
It's a bit odd by itself so I suggest we add context on what it's for:
```suggestion
// adapted from matlab, normalizes ____
export default function alpha_norm_type2(x1, x2, alpha) {
```

Reply:
done